.github/workflows/commit_check.yaml

```
@@ -3,65 +1,67 @@
-      DOPPLER_TOKEN: ${{ secrets.DOPPLER_TOKEN }}
-      DOPPLER_PROJECT: ${{ secrets.DOPPLER_PROJECT }}
-      DOPPLER_CONFIG: ${{ secrets.DOPPLER_CONFIG }}
     steps:
-      - name: Get Code Checkout
-        uses: 'Actions/Checkout@v3'
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+          cache: 'pip'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)